### PR TITLE
Fix duped load funcs

### PR DESCRIPTION
**Pull request by kaminow**
## Description
Small PR to remove some duplicated functions between `asapdiscovery.data.openeye` and `asapdiscovery.data.utils` and fix related import statements.

## Status
- [x] Ready to go

### CODE REVIEW

**Comment by apayne97** [comment]
@kaminow I had meant to fix this already, but I think we should have the openeye stuff in the openeye module? or else just get rid of that module

**Comment by kaminow** [comment]
ah yeah I think moving the functions into the `openeye` module probably makes more sense

File: asapdiscovery/data/fragalysis.py [hunk 1]
```diff
@@ -66,6 +66,8 @@ def parse_xtal(x_fn, x_dir):
     x_dir : str
         Path to directory containing directories with crystal structure PDB
         files
+    p_only : bool, default=True
+        Whether to filter to only -P* files in fragalysis
```
Comment (JenkeScheen) [inline asapdiscovery/data/fragalysis.py:70]: a quick explanation of what -P* files are would be helpful here

**Review by JenkeScheen — APPROVED** [review]
LGTM!

File: asapdiscovery/data/fragalysis.py (hunk 1 above)
Comment (kaminow) [reply asapdiscovery/data/fragalysis.py]: new comment in [d36aa85](https://github.com/choderalab/covid-moonshot-ml/pull/101/commits/d36aa85956f6936d00eff149accf7da0e8bac400) that I think should help

**Pull request merged**